docker-compose.yml

```diff
@@ -15,7 +15,9 @@ services:
   broker:
     image: rabbitmq
   celery:
-    build: .
+    build:
```

Comment:
``celery`` image is nothing more than a base python image with celery installed. In order to make it work you need to do complicated volume mounts to discover tasks. Also it doesn't support beat.